server/sched.go

```diff
@@ -143,6 +143,7 @@ func (s *Scheduler) processPending(ctx context.Context) {
 				slog.Debug("pending request cancelled or timed out, skipping scheduling")
 				continue
 			}
+			logutil.Trace("processing incoming request", "model", pending.model.ModelPath)
```

Comment:
I wonder if we should make these debug? (i.e. `OLLAMA_DEBUG=1`)? Trace may be too verbose to ask users for (and would more be for development debugging)